packages/chat-component/src/styles/chat-component.ts

```diff
@@ -81,131 +124,129 @@ export const chatStyle = css`
   }
   .headline {
-    color: var(--text-color);
-    font-size: 5vw;
+    color: var(--c-text);
+    font-size: var(--font-r-large);
     padding: 0;
-    margin: 10px 0 30px;
+    margin: var(--d-small) 0 var(--d-large);
 
     @media (min-width: 1024px) {
-      font-size: 3vw;
+      font-size: var(--font-r-base);
       text-align: center;
     }
   }
   .subheadline {
-    color: var(--text-color);
-    font-size: 1.2rem;
+    color: var(--c-text);
+    font-size: var(--font-r-base);
     padding: 0;
     margin: 0;
   }
+  .subheadline--small {
+    font-size: var(--font-small);
+    display: inline-block;
+  }
+  .chat__header--thread {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+  }
   .chat__container {
     min-width: 100%;
     transition: width 0.3s ease-in-out;
     max-height: 100vh;
   }
   .chat__containerWrapper.aside-open {
     .chat__listItem {
-      max-width: 90%;
-      min-width: 80%;
+      max-width: var(--width-wide);
     }
   }
   .chat__containerWrapper {
     display: grid;
     grid-template-columns: 1fr;
-    gutter: 20px;
+    gutter: var(--d-base);
   }
   .chat__containerWrapper.aside-open {
     display: grid;
     grid-template-columns: 1fr;
-    grid-column-gap: 20px;
-    grid-row-gap: 20px;
+    grid-column-gap: var(--d-base);
+    grid-row-gap: var(--d-base);
 
     @media (min-width: 1024px) {
       grid-template-columns: 1fr 1fr;
     }
   }
   .chat__containerWrapper.aside-open .aside {
     width: 100%;
-    border-left: 1px solid #d2d2d2;
+    border-left: var(--border-thin) solid var(--c-light-gray);
 
     @media (max-width: 1024px) {
-      width: 80%;
+      width: var(--width-base);
     }
   }
   @media (max-width: 1024px) {
     .aside {
-      top: 30px;
+      top: var(-d-large);
       left: auto;
       z-index: 3;
-      background: var(--white);
+      background: var(--c-white);
       display: block;
-      padding: 20px;
+      padding: var(--d-base);
       position: absolute;
-      width: 80%;
-      border-radius: 10px;
+      width: var(--width-base);
+      border-radius: var(--radius-base);
     }
   }
   .form__container {
-    margin-top: 30px;
-    padding: 10px;
+    margin-top: var(--d-large);
+    padding: var(--d-small);
   }
   .form__container-sticky {
     position: sticky;
     bottom: 0;
     z-index: 1;
-    border-radius: 10px;
-    background: linear-gradient(
-      0deg,
-      rgba(245, 245, 245, 1) 0%,
-      rgba(245, 245, 245, 0.8) 75%,
-      rgba(245, 245, 245, 0.5) 100%
-    );
-    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
-    padding: 15px 10px 50px;
+    border-radius: var(--radius-base);
+    background: linear-gradient(0deg, var(--c-base-gray) 0%, var(--c-base-gray) 75%, var(--c-base-gray) 100%);
+    box-shadow: var(--shadow);
+    padding: var(--d-small) var(--d-small) var(--d-large);
   }
   .form__label {
     display: block;
-    padding: 5px 0;
-    font-size: small;
-  }
-  .chat__header {
```

Comment:
merge issue?

Reply:
This is a new wrapper to better handle reusable header.